### ISSUE TITLE
Homework 1 - PyTorch Training Error

### ISSUE THREAD

**Issue by SaudxInu** [Luvata/CS330-Stanford-2019#33]
Hey,

I tried solving this in PyTorch, but my training loss is not improving.

I tried your PyTorch solution (copy and paste), same problem.

Am I missing something?

**Comment by Luvata** [comment]
Hi @SaudIqbal-IITM ,
Thank you for raising this issue. I wanted to let you know that I currently do not have the bandwidth to maintain this project at the moment. Additionally, the project is quite outdated and incomplete.
I apologize for any inconvenience this may cause. However, I hope to revisit it in the near future, possibly next month, when I have some free time available.
Thank you for your understanding.
